packages/@lwc/engine-core/package.json

```diff
@@ -17,7 +17,7 @@
     "license": "MIT",
     "scripts": {
         "clean": "rm -rf dist/ types/",
-        "build": "tsc --emitDeclarationOnly && rollup --config scripts/rollup.config.js"
+        "build": "rollup --config scripts/rollup.config.js && rm -fr types && mv dist/types types"
```

Comment:
What's the story with using `rm` vs a cross-platform package like https://www.npmjs.com/package/rimraf?

Reply:
AFAICT we have not made an effort to make any of our build processes work on Windows. There hasn't been a pressing need since none of the LWC devs use Windows.

IMO tools like rimraf, mkdirp, shx, cross-env, etc. are great if you have a team where someone is working on Windows. But if not, it's extra dependencies and complexity, and Windows compat is not tested in CI so it's fragile. Just my two cents.